.github/workflows/ms.aad.domainservices.yml

```diff
@@ -72,10 +71,15 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: Set environment variables
+        uses: ./.github/actions/templates/setEnvironmentVariables
+        with:
+          variablesPath: ${{ env.variablesPath }}
       - name: 'Run tests'
         uses: ./.github/actions/templates/validateModulePester
         with:
           modulePath: '${{ env.modulePath }}'
+          globalModuleTestFilePath: '${{ env.globalModuleTestFilePath }}'
```

Comment:
Not sure if I am a big fan if this. Adding additional parameters that take values from an action that wasn't invoked here before to set a value that is defined in yet another file makes it subjectively very hard to understand / trace back.
Not sure if it's worth the benefit of specfying the path in one place instead of 2. 

Would be curious what the other contributors/maintainers think :)